modules/check_in/app/serializers/check_in/vaos/appointment_serializer.rb

```diff
@@ -10,6 +10,29 @@ class AppointmentSerializer
 
       attributes :kind, :status, :serviceType, :locationId, :clinic, :start, :end, :minutesDuration
 
+      attribute :telehealth do |object|
+        {
+          vvsKind: object&.telehealth&.dig('vvsKind'),
+          atlas: object&.telehealth&.dig('atlas')
+        }
+      end
+
+      attribute :extension do |object|
+        {
+          preCheckinAllowed: object&.extension&.dig('preCheckinAllowed'),
+          eCheckinAllowed: object&.extension&.dig('eCheckinAllowed'),
+          patientHasMobileGfe: object&.extension&.dig('patientHasMobileGfe')
+        }
+      end
+
+      attribute :serviceCategory do |object|
+        object.serviceCategory&.map do |category|
+          {
+            text: category&.dig('text')
+          }
+        end
```

Comment:
can be simplified as:

```suggestion
        object.serviceCategory&.map { |category| { text: category['text'] } }
```